doc/source/getting_started/running_mechanical.rst

```diff
@@ -76,12 +76,17 @@ usage, type the following command:
         -h, --help                 Show this message and exit.
         -p, --project-file TEXT    Opens Mechanical project file (.mechdb). Cannot
                                     be mixed with -i
+        --private-appdata          Use a private appdata. This is beneficial for
```

Comment:
Make the appdata folder private. This enables you to run parallel instances of Mechanical.